### PR TITLE
Implement manage mode with off canvas block configure form.

### PR DESCRIPTION
**Pull request by yanniboi**

### CODE REVIEW

**Comment by yanniboi** [comment]
selection_257


selection_258

**Comment by andrewbelcher** [comment]
@yanniboi I think we need to get #68 in first due to edit links appear in the form...

File: js/dashboard.manage.js [hunk 1]
```diff
@@ -1,4 +1,40 @@
-(function ($, Drupal) {
+(function ($, Drupal, drupalSettings, _) {
```
Comment (andrewbelcher) [inline js/dashboard.manage.js:1]: What's the `_` variable for?

**Review by andrewbelcher — COMMENTED** [review]
Look pretty good, one question before it goes in!

File: js/dashboard.manage.js (hunk 1 above)
Comment (yanniboi) [reply js/dashboard.manage.js:1]: underscore. its a js library

**Pull request merged**